### PR TITLE
added pkr to noDivisionCurrencies

### LINKED ISSUE #4779: PKR currency is being divided by 100
### Describe the bug

PKR currency should not be divided by 100.

### System information

Medusa version (including plugins): 1.13.1
Node.js version: 18.16.0
Database: Postgres
Operating system: macOS Sonoma
Browser (if relevant): All

### Steps to reproduce the behaviour

1. Enable and Select PKR as the default currency.
2. Go to Medusa Admin > Products > New Product > Variants > Product Variants > Pricing, and add Price.
3. Change the region to PKR and go to the product page.
4. The price is divided by 100

### Expected behaviour

PKR currency should not divided by 100

### Screenshots

Screenshot 2023-08-16 at 5 54 22 PM
Screenshot 2023-08-16 at 5 54 56 PM

### PR DESCRIPTION
**Pull request by GroovierHen**
Fixes #4779

### CODE REVIEW

**Comment by srindom** [comment]
This file will have to be updated too: https://github.com/medusajs/medusa/blob/b08337deb7a9166175d3d3eab9e98172604b9b78/packages/medusa-core-utils/src/zero-decimal-currencies.ts#L4

**Comment by GroovierHen** [comment]
> This file will have to be updated too:
> 
> https://github.com/medusajs/medusa/blob/b08337deb7a9166175d3d3eab9e98172604b9b78/packages/medusa-core-utils/src/zero-decimal-currencies.ts#L4

Done

**Comment by GroovierHen** [comment]
@srindom can you tell me why the unit tests are failing so I can fix it?

File: packages/medusa-react/src/helpers/index.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ export const formatAmount = ({
 }
 
 // we should probably add a more extensive list
-const noDivisionCurrencies = ["krw", "jpy", "vnd"]
+const noDivisionCurrencies = ["pkr","krw", "jpy", "vnd"]
```
Comment (bacarybruno) [inline packages/medusa-react/src/helpers/index.ts:390]: Shouldn't this list be the exactly the same as the one in `packages/medusa-core-utils/src/zero-decimal-currencies.ts`?

Comment (GroovierHen) [reply packages/medusa-react/src/helpers/index.ts:390]: yeah, it needs to be.

Comment (bacarybruno) [reply packages/medusa-react/src/helpers/index.ts:390]: Do you think it's worth doing the change on this PR (importing and reusing `noDivisionCurrencies`) or it's out of scope? cc @srindom

**Comment by riqwan** [comment]
Hey, thanks for the PR! Since v2 brought a lot of architectural changes, we will be closing this PR since it no longer applies to our new setup.

If you would still like to open this PR, please rebase with the v1 branch and re-request a review & we'll get back on it quickly. If you would like to open this change for v2 as well, please reopen with the v2 branch as base.

**Pull request closed without merging**